File: src/codes/Users_Manual/segment/segment_change_transition.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Utils;
~using var autd = Controller.Open([new AUTD3()], new Nop());
autd.Send(SwapSegment.Modulation(Segment.S1, TransitionMode.Immediate));

File: src/codes/Users_Manual/segment/segment_change_transition.cpp
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
//~using namespace autd3;
//~auto autd =
//~Controller::open({AUTD3{}}, link::Nop{});
autd.send(SwapSegment::Modulation(Segment::S1, TransitionMode::Immediate()));
//~return 0; }